### ISSUE TITLE
Build Failure: AssertionError - "architecture" must be either "x86_64" or "arm64"

### ISSUE THREAD

**Issue by rajput-hemant** [AndrewBarba/bun-vercel#3]
Hey there,

I'm encountering an issue while attempting to deploy my project on Vercel using the Bun Runtime. The error message I'm receiving is:

```sh
$ bun-vercel ./src/main.ts
Build Completed in /vercel/output [1s]
AssertionError [ERR_ASSERTION]: "architecture" must be either "x86_64" or "arm64"
    at new Lambda (/var/task/sandbox.js:91:14706)
    at g7t (/var/task/sandbox.js:239:5093)
    at f3 (/var/task/sandbox.js:239:3640)
    at async TCe (/var/task/sandbox.js:245:4681)
    at async WBt (/var/task/sandbox.js:261:1990)
    at async zBt (/var/task/sandbox.js:261:1791) {
  generatedMessage: false,
  code: 'ERR_ASSERTION',
  actual: false,
  expected: true,
  operator: '=='
}
```
I've followed the setup instructions and attempted both the GitHub Integration and Manual methods, but this error persists.

**Comment by AndrewBarba** [comment]
Hey what OS are you using? Can you log process.arch in the node repl and paste it here?

**Comment by rajput-hemant** [comment]
@AndrewBarba 

Here,

![Screenshot](https://graph.org/file/9edd1de5c0b2f1bd2e7d2.png)

I did try to deploy after changing the architecture in the function config (`.vercel/output/functions/App.func`) to `x86_64` and `arm64`. Then I was presented with this error

![Screenshot](https://graph.org/file/98d31078947a8b6d0e454.png)

**Comment by rajput-hemant** [comment]
Hey there,

I hope you're doing well. I wanted to check if there any news or progress on this front? I'm eager to see if there have been any developments. If there's anything I can do to assist, please let me know.

Thanks!

**Comment by AndrewBarba** [comment]
@rajput-hemant I just published bun-vercel@1.0.0-alpha.5 which you can try
